setup.py

```diff
@@ -16,7 +16,7 @@
     packages=find_packages(exclude=('tests.*', 'tests', 'example')),
     install_requires=[
         'Django>=1.8',
-        'sqlparse',
+        'sqlparse>=0.2.0',
```

Comment:
@aaugustin 
https://requires.io/ could be used to track dependencies status. They provide a badge, which can be included in `README.md`, so if some package gets outdated it changes background from green to yellow or red alerting the issue.
They also list all packages with their versions like this: https://requires.io/github/celery/celery/requirements/?branch=5.0-devel
This way, it's easy to mention one requiring update.
There's also an option to be notified about outdated packages by email and/or pull request.